tools/oss-fuzz.sh

```diff
@@ -27,5 +27,10 @@ for d in "$(dirname $0)"/../tests/ts/fuzzers/test_*_fuzz_files; do
     zip -jqr $OUT/${fuzzer}_seed_corpus.zip "$d"
 done
 
+# create seed corpus for blkid fuzzing
+for d in "$(dirname $0)"/../tests/ts/blkid/images-fs; do
+    zip -jqr $OUT/test_blkid_fuzz_seed_corpus.zip "$d"
```

Comment:
The images in the source tree are compressed with `xz`. Shouldn't they be uncompressed before being added to the corpus?

What about also adding `images-pt`?

Reply:
Yes, thanks, libFuzzer accepts .xz files so I didn't even think much about it, but the performance difference at the start of fuzzing was noticeable, so uncompressed the images first.
Also added images-pt to the seed corpus